packages/cli/src/commands/generate/service/templates/service.ts.template

```diff
@@ -30,6 +30,6 @@ export const delete${singularPascalName}: MutationResolvers['delete${singularPas
   })
 }<% } %><% if (relations.length) { %>
 
-export const ${singularPascalName}: ${singularPascalName}Resolvers  = {<% relations.forEach(relation => { %>
-  ${relation}: (_obj, { root }) => db.${singularCamelName}.findUnique({ where: { id: root.id } }).${relation}(),<% }) %>
+export const ${singularPascalName}: Partial<${singularPascalName}Resolvers>  = {<% relations.forEach(relation => { %>
+  ${relation}: (_obj, gqlArgs) => db.${singularCamelName}.findUnique({ where: { id: gqlArgs?.root?.id } }).${relation}(),<% }) %>
```

Comment:
So, now we can access other arguments besides root.

See: https://graphql.org/learn/execution/#root-fields-resolvers

> args The arguments provided to the field in the GraphQL query.

Reply:
We could always do this, it was just desctructured before. Any thoughts on the name? Best I could come up with  was`gqlArgs`